<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialWindow</class>
 <widget class="QDialog" name="SerialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ControlWindow</string>
  </property>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/back.png</normaloff>img/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="SD">
    <attribute name="title">
     <string>SD</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>111</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ScanSD">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Scan SD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StartPrint">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Start Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="FileList">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>571</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="USB">
    <attribute name="title">
     <string>USB</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="ScanSD_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>100</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string>Active Print</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
